### PR TITLE
[Merged by Bors] - feat: port Data.Fintype.CardEmbedding

### PR DESCRIPTION
**Pull request by alexkassil**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by alexkassil** [comment]
Discussion here https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Addin.20.40alexkassil.20on.20github.20for.20port

File: Mathlib/Data/Fintype/CardEmbedding.lean [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+This file establishes the cardinality of `α ↪ β` in full generality.
+-/
+
+
+-- mathport name: finset.card
+local notation "|" x "|" => Finset.card x
+
+-- mathport name: fintype.card
+local notation "‖" x "‖" => Fintype.card x
```
Comment (dupuisf) [inline Mathlib/Data/Fintype/CardEmbedding.lean:26]:
```suggestion
local notation "|" x "|" => Finset.card x

local notation "‖" x "‖" => Fintype.card x
```

**Review by dupuisf — COMMENTED** [review]
Thanks!

bors d+

**Pull request closed without merging**